common/src/main/java/io/druid/metadata/MetadataStorageConnectorConfig.java

```diff
@@ -59,6 +59,7 @@ public int getPort()
     return port;
   }
 
+  @JsonProperty
```

Comment:
This missing `JsonProperty` looks a bug. Isn't it?

Reply:
I think it's not fixing any production bug, since the `@JsonProperty` on the `connectURI` field will get used if this one on the getter isn't there. In production that field is always going to be set. So I think this is technically a bug but it only affects test.